src/index.js

```diff
@@ -79,8 +79,9 @@ app.ports.startSequence.subscribe(function(seq){
   sequence = new Sequence(function(_, note){
     sampler.triggerAttackRelease(note, noteLength)
   }, seq, subdivision);
-  Transport.start()
+
   sequence.start();
+  Transport.start("+0.1")
```

Comment:
What is this?

Reply:
https://github.com/Tonejs/Tone.js/wiki/Performance#scheduling-in-advance